modules/core/src/main/java/org/apache/ignite/IgniteSystemProperties.java

```diff
@@ -58,30 +154,40 @@
      * @deprecated Use {@link #IGNITE_LOG_INSTANCE_NAME}.
      */
     @Deprecated
+    @SystemProperty(value = "If this system property is present Ignite will include grid name into " +
```

Comment:
hm, should we use GridGain instead of Ignite in messages?
@sergey-chugunov-1985  WDYT?